### PR TITLE
Update OWNERS for ART

### PR DESCRIPTION
**Pull request by huffmanca**
Includes the storage component for ART purposes.

### CODE REVIEW

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve
/ok-to-test

File: OWNERS [hunk 1]
```diff
@@ -5,3 +5,6 @@ reviewers:
 approvers:
 - andyzhangx
 - feiskyer
+
+component: "Storage"
+subcomponent: "Operators"
```
Comment (andyzhangx) [inline OWNERS:10]: why subcomponent is operators? do you have some reference link? thanks.

**Comment by huffmanca** [comment]
Apologies, this was for the wrong fork.

**Pull request closed without merging**